### ISSUE TITLE
Article on monitoring

### ISSUE THREAD

**Issue by RealOrangeOne** [selfhostedshow/wiki#33]
Monitoring your applications.

## Possible applications

- https://statping.com/
- https://www.nagios.org/
- https://icinga.com/

## Possible points

- Why you might _not_ want to selfhost all of this (yes, really!)

**Comment by patski123** [comment]
I agree that a section on why hosting monitoring applications externally is sometimes the better way to go.

**Comment by joeySeal** [comment]
I think discouraging people from "self hosting" might be a little broad. In regards to your monitoring what's more important than it being self hosted or not is if it is hosted in a way that allows it to remain up when the rest of your systems are down.

**Comment by Bas-Man** [comment]
I think something went wrong with my squash. This shouldn't be referenced. But I think it was just an incorrect hash+number. Sorry.
